src/supplemental/nanolib/conf.c

```diff
@@ -2388,7 +2392,12 @@ conf_bridge_node_parse_subs(
 			}
 		}
 	}
-
+	if (get_remote_topic) {
```

Comment:
I think it just fix the potential memleak mentioned in line 2297 and line 2308, right?